peerscout/keyword_extract/keyword_extract.py

```diff
@@ -240,12 +207,66 @@ def get_latest_state(
     return latest_timestamp
 
 
-def write_to_file(json_list, full_temp_file_location):
+def remove_cols_load_to_db_get_latest_timestamp(
```

Comment:
Maybe it would be slightly neater to separate the batch generation from the processing of an individual batch.
And maybe also the removing of the extra fields.

e.g.:
```python
data_with_extracted_keywords_batches = iter_get_batches(data_with_extracted_keywords, 3000)
for data_with_extracted_keywords_batch in data_with_extracted_keywords_batches:
     latest_timestamp = get_latest_timestamp(data_with_extracted_keywords_batch)
     write_to_jsonl_file(get_data_record_list(data_with_extracted_keywords_batch))
     upload_to_bigquery(...)
     update_state(latest_timestamp)
```